tests/nodescaling/README.md

```diff
@@ -0,0 +1,85 @@
+# Node scaling
+
+In the node scaling tests, the following workflow is followed:
+
+1. Provision a downstream cluster
+2. Perform post-cluster provisioning checks
+3. Scale the downstream cluster's nodes up to the desired amount
+4. Perform post-scaling up checks
+5. Scale the downstream cluster's nodes down to the desired amount
+6. Perform post-scaling down checks
+7. Cleanup resources (Terraform explicitly needs to call its cleanup method so that each test doesn't experience caching issues)
+
+Please see below for more details for your config. Please note that the config can be in either JSON or YAML (all examples are illustrated in YAML).
+
+## Table of Contents
+1. [Getting Started](#Getting-Started)
+2. [Scaling Node Pools](#Scaling-Node-Pools)
+
+## Getting Started
+In your config file, set the following:
+```yaml
+rancher:
+  host: "rancher_server_address"
+  adminToken: "rancher_admin_token"
+  insecure: true/optional
```

Comment:
is this truly optional?

Reply:
w.r.t. `insecure` ^^

Reply:
Yeah, that doesn't necessarily need to be there, but left it in there to signify it's available.